src/js/control-bar/mute-toggle.js

```diff
@@ -29,50 +29,51 @@ var MuteToggle = Button.extend({
       }
     });
   }
-});
 
-MuteToggle.prototype.createEl = function(){
-  return Button.prototype.createEl.call(this, 'div', {
-    className: 'vjs-mute-control vjs-control',
-    innerHTML: '<div><span class="vjs-control-text">' + this.localize('Mute') + '</span></div>'
-  });
-};
+  createEl() {
+    return Button.prototype.createEl.call(this, 'div', {
+      className: 'vjs-mute-control vjs-control',
+      innerHTML: '<div><span class="vjs-control-text">' + this.localize('Mute') + '</span></div>'
+    });
+  }
 
-MuteToggle.prototype.onClick = function(){
-  this.player_.muted( this.player_.muted() ? false : true );
-};
+  onClick() {
+    this.player_.muted( this.player_.muted() ? false : true );
+  }
 
-MuteToggle.prototype.update = function(){
-  var vol = this.player_.volume(),
-      level = 3;
+  update() {
+    var vol = this.player_.volume(),
+        level = 3;
 
-  if (vol === 0 || this.player_.muted()) {
-    level = 0;
-  } else if (vol < 0.33) {
-    level = 1;
-  } else if (vol < 0.67) {
-    level = 2;
-  }
+    if (vol === 0 || this.player_.muted()) {
```

Comment:
This doesn't translate to a switch since they're inequalities and not concrete values.
Also, switch is just a glorified if/else.
What about?
```js
level = (vol === 0)  ? 0 :
        (vol < 0.33) ? 1 :
        (vol < 0.67) ? 2 ;
```
or maybe
```js
if (vol === 0 || this.player_.muted()) {
  level 0;
} else
if (vol < 0.33) {
  level 1;
} else
if (vol < 0.67) {
  level = 2;
}
```
Or maybe
```js
if (vol === 0 || this.player_.muted()) {
  level = 0;                           } else
if (vol < 0.33) {
  level = 1;                           } else
if (vol < 0.67) {
  level = 2;                           }
```